studio-docs/source/schema-checks.mdx

```diff
@@ -88,6 +88,15 @@ You override flagged changes on an operation-by-operation basis. For each operat
 * **Ignore the operation.** In this case, schema checks will completely ignore the operation when checking _all_ changes in any future execution.
   * This option is useful when you know an operation originates _only_ from clients or client versions that you don't actively support.
 
+### Rerunning a check
+
+You can also rerun checks from inside Studio. When you rerun a check, the latest check configuration will be applied.
+The new check will use an updated time window for operations — in addition, any changes made to excluded / included clients, checked variants, and any operations already marked safe / ignored will be incorporated when rerunning.
```

Comment:
Merge this line with the previous one (we keep each paragraph on a single line to reduce diff volatility when editing a paragraph)

"will use" -> "uses".

Replace em dash with period

"will be" -> "are"